frappe/public/js/lib/leaflet/leaflet.css

```diff
@@ -11,9 +12,9 @@
 .leaflet-image-layer,
 .leaflet-layer {
 	position: absolute;
-	left: 0;

```

Comment:
@alaa-alsalehi reintroduce this as well. It is required

Reply:
done